pkg/controller/storagecluster/cephobjectstores_test.go

```diff
@@ -1,21 +1,38 @@
 package storagecluster
 
 import (
+	"testing"
+
 	api "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
 	"github.com/openshift/ocs-operator/pkg/controller/defaults"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	"testing"
 )
 
 func TestCephObjectStores(t *testing.T) {
+	var cases = []struct {
+		label                string
```

Comment:
wouldn't "name" be a better variable to denote Case name?

Reply:
Every test file uses the `label` variable to denote case name. I have added a comment beside the case. Let me know if other changes are required.